omnibus/config/software/datadog-agent.rb

```diff
@@ -237,4 +239,36 @@
   unless windows_target?
     delete "#{install_dir}/uselessfile"
   end
+
+  # TODO: move this to omnibus-ruby::health-check.rb
+  # check that linux binaries contains OpenSSL symbols when building to support FIPS
+  if fips_mode? && linux_target?
```

Comment:
Could you clarify the differences between this block and `inspect_binary(bin, &raise_if_fips_symbol_not_found)` above?

Reply:
They are doing the same job, but I was not familiar with why we `inspect()` the functions in a `def install_sources` function so I was not confident to use it for Linux as well. But actually I think I would be able to use `inspect_binary` for Linux as well (I'll give a try)

Reply:
For some reason if I use `inspect_symbols` here the function is not called. I'll leave it as it for now